File: articles/proxy-variables-and-measurement-error/go/proxy_measurement_risk.go
package main

import "fmt"

func main() {
	validityGap := 0.42
	missingness := 0.12
	differentialError := 0.24
	labelError := 0.08
	score := (validityGap + missingness + differentialError + labelError) / 4.0
	fmt.Printf("measurement_risk_score=%.4f\n", score)
}

File: articles/proxy-variables-and-measurement-error/typescript/proxy_measurement_risk.ts
const validityGap = 0.42;
const missingness = 0.12;
const differentialError = 0.24;
const labelError = 0.08;
const score = (validityGap + missingness + differentialError + labelError) / 4.0;
console.log(`measurement_risk_score=${score.toFixed(4)}`);
